core/services/gas_updater.go

```diff
@@ -31,8 +31,6 @@ var (
 	)
 )
 
-//go:generate mockery -name GasUpdater  -output ../internal/mocks/ -case=underscore
```

Comment:
Why was this removed?

Reply:
It's unused, and its output is not checked in.